phys/module_fr_fire_atm.F

```diff
@@ -33,10 +34,23 @@ subroutine add_fire_tracer_emissions(    &
 real,intent(in)::rho(ims:ime,kms:kme,jms:jme),dz8w(ims:ime,kms:kme,jms:jme)
 real,intent(in),dimension(ifms:ifme,jfms:jfme)::burnt_area_dt,fgip
 real,intent(inout)::tracer(ims:ime,kms:kme,jms:jme,num_tracer)
+
+integer, intent(in) :: fire_smk_scheme !switch for smoke release
+real, intent(in) :: fire_smk_peak !peak smoke release height for TG
+real, intent(in) :: fire_smk_ext  !smoke extinction depth for TG
+real, intent(in) :: fire_tg_ub    !upper bound of TG
+real, intent(in), dimension( ims:ime,kms:kme,jms:jme ) :: z_at_w ! m abv sealvl
+real, intent(in), dimension( ims:ime,jms:jme ) :: zs  ! topography (m abv sealvl)
+
 ! local
 integer::isz1,jsz1,isz2,jsz2,ir,jr
 integer::i,j,ibase,jbase,i_f,ioff,j_f,joff
 real::avgw,emis,conv
+integer :: i_st,i_en,j_st,j_en
+
+!local for TG
+integer :: k,k_st,k_en
+real, dimension(ims:ime,kms:kme,jms:jme) :: prop_smk
```

Comment:
If prop_smk is a local array, it is better to define it in the tiles dimensions

Reply:
changed.